test/ttmlir/Dialect/TTNN/simple_squeeze.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline %s| FileCheck %s
+#any_device_tile = #tt.operand_constraint<dram|l1|tile|any_device_tile>
+module attributes {} {
+  func.func @forward(%arg0: tensor<1x2x1x32x32xbf16>) -> tensor<1x2x32x32xbf16> {
```

Comment:
As a note, consider testing cases with rank 2, 3 as well as those can appear in models commonly

Reply:
Yea I added issue #549 to add more testing for `concat`/`reshape`/`squeeze`/`unsqueeze`

Reply:
Thanks! I'll let you know guys if we spot some specific cases that are required for Llama :))